reference/config_files/artifacts.properties.rst

```diff
@@ -3,17 +3,24 @@
 artifacts.properties
 ====================
 
-This file is used to send custom headers in the PUT requests that :command:`conan upload` command does:
+This is a file in the Conan cache that is useful to define a set of key-value pairs that will
+be sent together with the packages uploaded in the :command:`conan upload` command.
+This information is sent as custom headers in the PUT request and, if the server has the
+capability, as `matrix params`_.
+
 
 **.conan/artifacts.properties**
 
 .. code-block:: text
 
    custom_header1=Value1
    custom_header2=45
+   build.name=BuildJob
+
 
-Artifactory users can use this file to set file properties for the uploaded files. The variables should have the prefix
-``artifact_property``. You can use ``;`` to set multiple values to a property:
+Artifactory users can benefit from this capability to set file properties for the uploaded files.
+If the Artifactory version doesn't support matrix params yet (available since 6.XX) it will use
```

Comment:
I don't know the Artifactory release for this feature 😓

Reply:
Yes. we have to wait.

Reply:
Add a "request changes" review, so we don't merge this one accidentally